services/containers/local.go

```diff
@@ -109,8 +111,14 @@ func (l *local) ListStream(ctx context.Context, req *api.ListContainersRequest,
 }
 
 func (l *local) Create(ctx context.Context, req *api.CreateContainerRequest, _ ...grpc.CallOption) (*api.CreateContainerResponse, error) {
+	span := trace.SpanFromContext(ctx)
+	_, ctrCreateSpan := (span.Tracer().Start(ctx, "ctrCreateSpan"))
+
+	defer ctrCreateSpan.End()
+
 	var resp api.CreateContainerResponse
 
+	ctrCreateSpan.AddEvent("container create", trace.WithAttributes(attribute.String("containerd_id", req.Container.ID), attribute.String("container_image", req.Container.Image)))
```

Comment:
I might suggest putting these attributes on the span itself.  Since this event is created immediately after the span is started and before any work is done its timing can't really convey any information.